lib/router.js

```diff
@@ -367,6 +367,7 @@ Router.prototype.find = function find(req, res, callback) {
         }
     }
 
+    var maxV;
```

Comment:
Can you move this up to the top of the function, with the rest of the variable declarations?